config/locales/en.yml

```diff
@@ -3701,12 +3701,9 @@ en:
     default_sequence_name: "New sequence"
     edit_sequence: "Edit in Sequence editor"
     sequence_name_placeholder: "Click here to enter sequence name"
-    promo:
-      title: "Manage protein sequences in SciNote:"
-      bullet_1: "Import"
-      bullet_2: "Visualize"
-      bullet_3: "Annotate"
-      button: "Try Sequence editor"
+    trial_expiration_warning_html:
```

Comment:
Please remove this here. This will be done in another ticket with the correct description and links.